examples/spectralLinearElastic2MaterialsTM.i

```diff
@@ -0,0 +1,246 @@
+[GlobalParams]
+  displacements = 'u_x u_y u_z'
+[]
+
+[Mesh]
+  [generated_mesh]
+    type = GeneratedMeshGenerator
+    dim = 3
+    xmin = -5
+    xmax = 5
+    ymin = -5
+    ymax = 5
+    zmin = -5
+    zmax = 5
+    nx = 32
+    ny = 32
+    nz = 32
+  []
+
+  [cnode]
+    type = ExtraNodesetGenerator
+    coord = '0 0 0'
+    new_boundary = 100
+    input = generated_mesh
+  []
+[]
+
+[AuxVariables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+  [./disp_z]
+  [../]
+
+[./stress_xy]
+  order = CONSTANT
+  family = MONOMIAL
+[../]
+[./elastic_strain_xy]
+  order = CONSTANT
+  family = MONOMIAL
+[../]
+
+[]
+
+[Variables]
+  [./u_x]
+  [../]
+  [./u_y]
+  [../]
+  [./u_z]
+  [../]
+  [./global_strain]
+    order = SIXTH
+    family = SCALAR
+  [../]
+[]
+
+[Kernels]
+  [./TensorMechanics]
+  [../]
+[]
+
+[Functions]
+  [./young_modulus_function]
+    type = ParsedFunction
+    vars = 'radius ym_min ym_max'
+    vals = '2.0      1.0  4.0'
+    value = 'if(sqrt(x*x+y*y+z*z)<radius,ym_min,ym_max)'
+  [../]
+[]
+
+[ScalarKernels]
+  [./global_strain]
+    type = GlobalStrain
+    variable = global_strain
+    global_strain_uo = global_strain_uo
+  [../]
+[]
+
+[Materials]
+  [./strain]
+    type = ComputeSmallStrain
+    global_strain = global_strain
+  [../]
+  [./elasticity_tensor]
+    type = ComputeIsotropicElasticityTensor
+    youngs_modulus = 1.0e4
+    poissons_ratio = 0.3
+    elasticity_tensor_prefactor = young_modulus_function
+    # Is the line above during the righ thing by scaling the young modulus?
```

Comment:
```suggestion
    # Is the line above during the right thing by scaling the young modulus?
```

Reply:
Yes

Reply:
That comment shouldn't be there anymore...